gig-marketplace.md

```diff
@@ -0,0 +1,58 @@
+## Title
+* Create an intranet marketplace where InnerSource project owners and potential contributors can easily discover one another
+
+## Patlet
+Establish a marketplace by creating a website(s) that lists InnerSource project needs as "gigs". 
+You can attract potential contributors by posting gigs with explicit requirements (amount of time, skills). People can sign up for a gig and garner approval from their management for those explicit requirements. Individual contributors are then awarded points for the gigs they complete. 
+This makes it explicit what the commitment is and that management agrees with it while encouraging and tracking involvement.
+
+## Problem
+There is no single, uniform way for InnerSource project owners and potential contributors to discover each other and connect.
+
+## Story (Optional)
+Your InnerSource program has been established and there is great interest but little actual activity. 
+Individual project owners are encountering difficulties attracting a community of contributors.
+Most managers are supportive in allowing their employees the time to involve themselves in an InnerSource project. 
+There simply is no single uniform way for contributors to discover all the InnerSource opportunities that exist that they might be interested in contributing to.
+Conversely, innerSource project owners also have no way of informing a general audience about their projects and its goals.
+
+## Context
+* There are multiple business units in your organization that do not communicate well.
+* Individuals have no way of discovering what InnerSource opportunities exist across the entire company.
+* InnerSource project owners have no way of advertising their project to potential contributors.
+* Individuals have been given some time by their management to get involved in InnerSource projects of their choosing.
+* Managers have no way of knowing what the true time/effort commitment will be if their employee chooses to take on tasks to support an InnerSource project.
+* There is no way of tracking or rewarding an individual's involvement in an InnerSource project.
+
+## Forces
```

Comment:
Recommend rephrasing some of these lines to reference why the particular activity is important instead of just saying that the activity isn't happening.

e.g. `Managers have no way of understanding what the time/effort commitment is for their employees wanting to get involved in an InnerSource project outside of their business unit` becomes sometime like `Part of a manager's job is tracking, estimating and understanding use of employee time`.

Reply:
True.  Good point.  We should also edit this to include more of the motivations behind why individuals would want to take on a Gig.  To learn a new skill, demonstrate expertise, help someone out etc...  Right now it is too "manager"-centric.  The Gig Marketplace is actually something that benefits individual employees from a professional development perspective more than it does managers looking for assistance in getting day to day work completed.

Reply:
Sounds good.  Looking forward to continuing to work on this pattern together!